server/src/com/treode/server/SchematicStore.scala

```diff
@@ -22,6 +26,43 @@ class SchematicStore (store: Store, schema: Schema) {
     store.write (tx, ct, ops:_*)
   }
 
+  def batch_write(tx: TxId, ct: TxClock, req: Request): Async [TxClock] = {
```

Comment:
Move line 31 from here into the Resource handler, and pass in the JSON node instead of the whole request.